app/controllers/users_controller.rb

```diff
@@ -1,40 +1,42 @@
+# frozen_string_literal: true
+
 class UsersController < ApplicationController
   attr_accessor :remember_token
 
   def index
     @user = User.all
-  end 
```

Comment:
You never really get all users so this may not be needed

Reply:
Ok will remove that